src/safeposix/filesystem.rs

```diff
@@ -636,22 +636,3 @@ pub fn incref_root() {
         panic!("Root directory inode was not a directory");
     }
 }
-
-pub fn decref_dir(cwd_container: &interface::RustPathBuf) {
```

Comment:
is this function just not necessary anymore?

Reply:
it is used in two cases: when calling exit_syscall() to be able to delete the current working directory of the process we are killing and when calling chdir_syscall() to be able to close an old current working directory. The mechanism of incrementing the refcount of the cwd is only used in chdir_syscall(), and since it is removed in this PR, there is no need for decrementing the refcount either.